grafana/data_source_organization.go

```diff
@@ -38,6 +63,28 @@ func dataSourceOrganizationRead(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 
+	orgUsers, err := client.OrgUsers(org.ID)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	userCollections := map[string][]string{
+		"admins":  []string{},
+		"editors": []string{},
+		"viewers": []string{},
+	}
+
+	for _, user := range orgUsers {
+		role := fmt.Sprintf("%ss", strings.ToLower(user.Role))
+		userCollections[role] = append(userCollections[role], user.Email)
+	}
+
+	for roleCollection, emails := range userCollections {
```

Comment:
```suggestion
	for role, emails := range userCollections {
```

It's just a role, not a collection of it

Reply:
@julienduchesne I believe `roleCollection` refers to the key name of the `map[string][]string` and is plural (i.e. `admins`, `editors`, `viewers`). Nonethless, I'm happy to change this variable name if you prefer.